packages/application-extension/schema/commands.json

```diff
@@ -47,6 +47,11 @@
       "keys": [""],
       "selector": "body"
     },
+    {
+      "command": "application:toggle-fullscreen-mode",
+      "keys": ["Alt F"],
```

Comment:
`Alt + F` is used by browsers to:
- open File menu (Firefox)
- open three dots menu (Chrome/Edge)

It may or may not be ok to override these. We can discuss this on JupyterLab call on Wednesday. Would it be also ok to just use F11 which is typically default for full screen? This way the contribution of this PR is:
- it adds a menu entry
- it allows user to change the shortcut
- it does not force a no-standard shortcut which is normally used for something else